### DISCUSSION TITLE
安装问题，有没有人遇到，通过源码安装是空白页，npm安装报错

### DISCUSSION THREAD

**Discussion by YangYue2022** [ppoffice/hexo-theme-icarus#1326] [Q&A]
PS D:\> hexo init blog
INFO  Cloning hexo-starter https://github.com/hexojs/hexo-starter.git
INFO  Install dependencies
INFO  Start blogging with Hexo!
PS D:\> cd blog
PS D:\blog> npm install -S hexo-theme-icarus hexo-renderer-inferno
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: basic-auth@2.0.1
npm warn Found: peer eslint-plugin-standard@">=4.0.0" from eslint-config-standard@12.0.0
npm warn node_modules/.pnpm/basic-auth@2.0.1/node_modules/basic-auth/node_modules/eslint-config-standard
npm warn   dev eslint-config-standard@"12.0.0" from basic-auth@2.0.1
npm warn   node_modules/.pnpm/basic-auth@2.0.1/node_modules/basic-auth
npm warn
npm warn Could not resolve dependency:
npm warn peer eslint-plugin-standard@">=4.0.0" from eslint-config-standard@12.0.0
npm warn node_modules/.pnpm/basic-auth@2.0.1/node_modules/basic-auth/node_modules/eslint-config-standard
npm warn   dev eslint-config-standard@"12.0.0" from basic-auth@2.0.1
npm warn   node_modules/.pnpm/basic-auth@2.0.1/node_modules/basic-auth
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: compressible@2.0.18
npm warn Found: peer eslint-plugin-standard@">=4.0.0" from eslint-config-standard@14.1.0
npm warn node_modules/.pnpm/compressible@2.0.18/node_modules/compressible/node_modules/eslint-config-standard
npm warn   dev eslint-config-standard@"14.1.0" from compressible@2.0.18
npm warn   node_modules/.pnpm/compressible@2.0.18/node_modules/compressible
npm warn
npm warn Could not resolve dependency:
npm warn peer eslint-plugin-standard@">=4.0.0" from eslint-config-standard@14.1.0
npm warn node_modules/.pnpm/compressible@2.0.18/node_modules/compressible/node_modules/eslint-config-standard
npm warn   dev eslint-config-standard@"14.1.0" from compressible@2.0.18
npm warn   node_modules/.pnpm/compressible@2.0.18/node_modules/compressible
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: finalhandler@1.1.2
npm warn Found: peer eslint-plugin-standard@">=4.0.0" from eslint-config-standard@12.0.0
npm warn node_modules/.pnpm/finalhandler@1.1.2/node_modules/finalhandler/node_modules/eslint-config-standard
npm warn   dev eslint-config-standard@"12.0.0" from finalhandler@1.1.2
npm warn   node_modules/.pnpm/finalhandler@1.1.2/node_modules/finalhandler
npm warn
npm warn Could not resolve dependency:
npm warn peer eslint-plugin-standard@">=4.0.0" from eslint-config-standard@12.0.0
npm warn node_modules/.pnpm/finalhandler@1.1.2/node_modules/finalhandler/node_modules/eslint-config-standard
npm warn   dev eslint-config-standard@"12.0.0" from finalhandler@1.1.2
npm warn   node_modules/.pnpm/finalhandler@1.1.2/node_modules/finalhandler
npm error code EUNSUPPORTEDPROTOCOL
npm error Unsupported URL Type "link:": link:./scripts/eslint-plugin
npm error A complete log of this run can be found in: E:\Environmet\Nodejs\node_cache\_logs\2024-11-04T08_11_49_194Z-debug-0.log

**Comment by ppoffice** [comment]
你的Node.js版本是什么？看起来很多依赖都没有正确安装。
